core/driver/sources/src/test/scala/org/virtuslab/ideprobe/dependencies/IdeProbeConfigTest.scala

```diff
@@ -0,0 +1,100 @@
+package org.virtuslab.ideprobe.dependencies
+
+import scala.concurrent.duration._
+
+import org.virtuslab.ideprobe.Config
+import org.virtuslab.ideprobe.IdeProbeFixture
+import org.virtuslab.ideprobe.config.IntellijConfig
+import org.virtuslab.ideprobe.config.WorkspaceConfig
+
+import org.junit.Assert._
+import org.junit.Test
+import org.virtuslab.ideprobe.IntelliJFixture
+
+class IdeProbeConfigTest extends IdeProbeFixture {
+  private val configRoot = "probe" // same as in object IntelliJFixture
+
+  @Test
```

Comment:
Wow there's no scalatest in this project, just JUnit? 😯

Reply:
(I'm just surprised, I don't think it's really worth migrating that much if everything is already in JUnit)

Reply:
Yes, other tests use JUnit (at least in this module).